packages/contracts-bedrock/scripts/Deployer.sol

```diff
@@ -427,6 +438,36 @@ abstract contract Deployer is Script {
         return string(res);
     }
 
+    /// @dev Pulls the `_initialized` storage slot information from the Forge artifacts for a given contract.
+    function getInitializedSlot(string memory _contractName) internal returns (StorageSlot memory slot_) {
+        string memory storageLayout = getStorageLayout(_contractName);
+
+        string[] memory command = new string[](3);
+        command[0] = Executables.bash;
+        command[1] = "-c";
+        command[2] = string.concat(
```

Comment:
A neckbeard someplace would say this is a useless use of echo when you can also use `<` but its not that big of a deal - https://porkmail.org/era/unix/award